src/pages/Reservation/ReservUserInfo/ReservUserInfo.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PassengerInfo from './PassengerInfo/PassengerInfo';
+
+export default function ReservUserInfo() {
+  const [bookerName, setBookerName] = useState('');
+  const [bookerEmail, setBookerEmail] = useState('');
+  const [bookerNumber, setBookerNumber] = useState(0);
+
+  const validateName =
+    bookerName.length > 1 || bookerName === '' ? true : false;
+
+  const validateEmail =
+    bookerEmail.includes('@' && '.com') || bookerEmail === '' ? true : false;
+
+  const validateNumber =
+    bookerNumber.toString().length > 9 || bookerNumber === 0 ? true : false;
+
+  return (
+    <ReservUserInfoWrapper>
```

Comment:
```suggestion
    <Container>
```
가장 바깥 쪽 요소는 그냥 Container로 통일해서 만들어도 좋을거 같습니다